.nvmrc

```diff
@@ -0,0 +1 @@
+v16.15.0
```

Comment:
Should we include say this is the minimum version? Ie. does .nvmrc  ^16.15.0 or ~16.15.0?